### PR TITLE
add draft option

### PR DESCRIPTION
**Pull request by ririsoft**
Hello,

Thank you for your gh actions !

Please find through this PR a suggestion to add 'draft' input option to create GH releases as draft.
I hope it helps.

Cheers.

### CODE REVIEW

File: main.sh [hunk 1]
```diff
@@ -62,5 +63,12 @@ if gh release view "${tag}" &>/dev/null; then
     # https://cli.github.com/manual/gh_release_delete
     gh release delete "${tag}" -y
 fi
+
+gh_options=""
+
+if [[ "$draft" == "1" ]]; then
```
Comment (taiki-e) [inline main.sh:69]: Could you change this option to accept `true` and `false` instead of `0` and `1`?

**Review by taiki-e — CHANGES_REQUESTED** [review]
Thanks for the PR! This seems to make sense.

File: main.sh (hunk 1 above)
Comment (ririsoft) [reply main.sh]: The option now accepts `true` for draft mode and anything else is considered as standard release mode. There are no `--no-draft` or `--release` option that makes parsing `false` useful unfortunately.

**Comment by taiki-e** [comment]
Could you update the documentation as well?

**Comment by ririsoft** [comment]
> 
> 
> Could you update the documentation as well?

My bad, this is fixed in my last forced push, sorry.

File: action.yml [hunk 2]
```diff
@@ -9,6 +9,10 @@ inputs:
     description: Format of title (variables `$tag`, `$version`, and any string)
     required: false
     default: '$tag'
+  draft:
+    description: Create a draft release ('0' or '1')
+    required: false
+    default: '0'
```
Comment (taiki-e) [inline action.yml:15]:
```suggestion
    default: 'false'
```

**Review by taiki-e — APPROVED** [review]
Thanks!

**Comment by ririsoft** [comment]
> 
> 
> Thanks!

Thanks for your time, patience and work here.

**Comment by taiki-e** [comment]
Published in v1.1.0.

**Pull request merged**